src/webgpu/api/validation/encoding/cmds/render/indirect_multi_draw.spec.ts

```diff
@@ -112,7 +128,7 @@ g.test('buffers,device_mismatch')
       // eslint-disable-next-line @typescript-eslint/no-explicit-any
       (encoder as any).multiDrawIndirect(indirectBuffer, 0, 1, drawCountBuffer);
     }
-    validateFinish(!mismatched);
+    validateFinish(!indirectMismatched && (!useDrawCountBuffer || !drawCountMismatched));
```

Comment:
Ah I forgot also that drawCountMismatched wouldn't matter if !useDrawCountBuffer. That useless case should ideally be filtered out as well.

Reply:
Done.